docs/manual_install_guide_hpe_3par_plugin_with_openshift_kubernetes.md

```diff
@@ -263,6 +263,41 @@ $ cp /root/.kube/config /etc/kubernetes/admin.conf
 
 >Re-run the command to start the HPE 3PAR FlexVolume dynamic provisioner
 
+>**NOTE:** For multi-master cluster follow the below steps:
+
+>HPE 3PAR FlexVolume dynamic provisioner on OpenShift 3.11 multi-master cluster:
+```
+Run the below command to grant service account access to the hostPath plugin and the ability to run the container as root
+$ oc adm policy add-scc-to-user privileged system:serviceaccount:kube-system:doryd
+
+Run the below command to deploy doryd
+$ wget https://raw.githubusercontent.com/hpe-storage/python-hpedockerplugin/master/provisioner/OpenShift/dep-kube-storage-controller-ocp311.yaml
+$ oc create -f dep-kube-storage-controller-ocp311.yaml
+
+Run the below command to verify whether doryd is deployed successfully
+$ oc get deploy --namespace kube-system –o wide
+
```

Comment:
-  You can add a `kubectl get pods -n kube-system` to show the successful deployment of the doryd
-  plus add a `kubectl logs ` statement for the how to retrieve the provisioner logs.

Reply:
@wdurairaj Done.